### DISCUSSION TITLE
Can't do pr

### DISCUSSION THREAD

**Discussion by AbrarNoob** [ROM-builders/temporary#8931] [Ideas]
Today i tried to add support for AOSP-Krypton by pr..but i saw can't pr as merging blocked 
![Screenshot_20220420-111234_Chrome](https://user-images.githubusercontent.com/70853665/164155656-4f80c528-570e-432c-8288-e481a7540893.png)

**Comment by Apon77** [comment]
> Today i tried to add support for AOSP-Krypton by pr..but i saw can't pr as merging blocked ![Screenshot_20220420-111234_Chrome](https://user-images.githubusercontent.com/70853665/164155656-4f80c528-570e-432c-8288-e481a7540893.png)

You can't merge pr. Only I can. You can only do pr and request me to merge.

**Comment by Apon77** [comment]
Also, you cant build krypton without ssh keys. which is complex. So, you wont be able to sync the source in our ci because of not using ssh
